src/codelabs/async-await.md

```diff
@@ -233,12 +233,12 @@ your window is wide enough — is to the right of the synchronous example.
 
 ```dart
 // Synchronous
-String createOrderMessage() {
-  var order = getUserOrder();
+String printOrderMessage() {
+  var order = fetchUserOrder();
   return 'Your order is: $order';
 }
```

Comment:
This is the function definition of `createOrderMessage()` that should remain named `createOrderMessage()` and that I was referring to my previous comment.

Reply:
Btw, the _other_ definition of `createOrderMessage()` is currently only found in the following Gist: https://gist.github.com/d7abfdea1ae5596e96c7c0203d975dba